addons/addon-base-raas/packages/base-raas-services/lib/budgets/budgets-service.js

```diff
@@ -225,13 +233,26 @@ class BudgetsService extends Service {
   async _getBudgetClientAndAWSAccountId(requestContext, id) {
     const { accessKeyId, secretAccessKey, sessionToken, accountId } = await this._getCredentials(requestContext, id);
     const aws = await this.service('aws');
-    const budgetsClient = new aws.sdk.Budgets({
-      apiVersion: '2017-10-25',
-      region: 'us-east-1',
-      accessKeyId,
-      secretAccessKey,
-      sessionToken,
-    });
+    const region = this.settings.get(settingKeys.awsRegion);
+    let budgetsClient;
+    if (region === 'cn-northwest-1' || region === 'cn-north-1') {
+      budgetsClient = new aws.sdk.Budgets({
+        apiVersion: '2017-10-25',
+        region,
+        endpoint: 'budgets.amazonaws.com.cn',
```

Comment:
if it's required, pls create variables for endpoint and region, moving budgetClient out of if...else clause

Reply:
Yes, refactored